lib/media/streaming_engine.js

```diff
@@ -1708,6 +1732,54 @@ shaka.media.StreamingEngine = class {
     }
   }
 
+  /**
+   * Parse PRFT box.
+   * @param {!shaka.media.SegmentReference} reference
+   * @param {!shaka.extern.ParsedBox} box
+   * @private
+   */
+  parsePrft_(reference, box) {
+    if (this.parsedPrftEventRaised_) {
+      return;
+    }
+    box.reader.readUint32(); // Ignore referenceTrackId
+    const ntpTimestampSec = box.reader.readUint32();
+    const ntpTimestampFrac = box.reader.readUint32();
+    const ntpTimestamp = ntpTimestampSec * 1000 + ntpTimestampFrac / 2**32;
+
+    let mediaTime;
+    if ( box.version === 0) {
+      mediaTime = box.reader.readUint32();
+    } else {
+      mediaTime = box.reader.readUint64();
```

Comment:
You may get JS_INTEGER_OVERFLOW error

Reply:
Thanks for your input! 
I have added a check and a test for this scenario.